CONST_vars.yaml

```diff
@@ -231,7 +235,7 @@ vars:
     # Checker configuration
     checker:
         fulltextsearch: text to search
-        lang_files: [ngeo, cgxp, cgxp-api]
+        lang_files: [cgxp, cgxp-api]
```

Comment:
c2cgeoportal#master [does](https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl#L248) have `ngeo` in the `lang_files`. But c2cgeoportal#1.6 does [not](https://github.com/camptocamp/c2cgeoportal/blob/1.6/c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl#L241). @sbrunner, will it be a problem to have it in 1.6?